pkg/api/aim/query/query.go

```diff
@@ -38,12 +39,20 @@ type parsedQuery struct {
 	conditions []clause.Expression
 }
 
-type attributeGetter func(attr string) (any, error)
-
 type callable func(args []ast.Expr) (any, error)
 
+var negativeClause = func(expression clause.Expression) clause.Expression {
```

Comment:
why do you make it a func variable and not directly a func? 🤔

Reply:
changed in this style.